src/guides/about-multiprotocol.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: guide
+guideName: About multiprotocol support
+---
+
+<div class="container">
+  <h2>Introduction</h2>
+  <p><b>Note</b>: Multiprotocol is an <b>experimental</b> feature. It may not work for you. If it does not work, you may need to disable multi-protocol and you might even need to re-pair your devices.</p>
+  <p>
+    <span
+      >
+      Home Assistant Yellow's integrated wireless chip supports Zigbee by default.
+      By updating the firmware, the wireless chip can be used in a multiprotocol
+      setup supporting Zigbee and Thread at the same time! There are two
+      limitations: The two networks have to be on the same channel and share the
+      bandwidth. This page explains the background.</span
+    >
+  </p>
+  <h2>About channels</h2>
+  <p><span>Both Zigbee and Thread are low-rate wireless personal area networks
+    (LR-WPAN, IEEE 802.15.4). The standard specifies 16 channels in the 2.4&nbsp;GHz
+    spectrum (channel number 11-25). The standard supports multiple networks on
+    the same channel. Each network is identified by a PAN-ID (personal area network
+    identifier).
+  </span></p>
+  <p><span>
+    The wireless chip in Home Assistant Yellow supports all 16 channels. However,
+    it can only communicate on one single channel at a time. Due to this limitation
+    it is crucial that Zigbee and Thread run on the same channel for reliable
+    operation!
+    
+    Thanks to the PAN-ID, the Zigbee and Thread networks are still separated and
+    therefore won't interfere with each other.
+  </span></p>
+  <h2>About bandwidth</h2>
+  <p><span>
+    IEEE 802.15.4 (so both Zigbee and Thread) support a transfer rate of up to
+    250&nbsp;kbit/s. For typical Zigbee and Thread networks this is sufficient.
+    However, in the multiprotocol setup Zigbee and Thread networks need to share
+    the same channel, and therefor share the 250&nbsp;kbit/s transfer rate. Furthermore,
+    if Wi-Fi or other 2.4&nbsp;GHz networks are running in a similar frequency range,
+    transfer rate might be further limited.
+  </span></p>
+  <p><span>
+    For large networks it is advisable to add a second wireless chip. This allows
+    to run Zigbee and Thread each on a dedicated wireless chip and on separate
+    channels.
+  </span></p>
+  <h2>Resolve channel conflict</h2>
+  <p>You may get a notification stating that the OTBR and ZHA share the same  radio, but use different channels. This mode of operation may cause an unreliable network connection. To resolve this issue, you can try one of the following methods: 
+    <p><b>Option 1</b>: Reset the Thread network by using the <b>Reset border router</b> button. <img src="/images/multiprotocol-reset-thread-network.png"/><ul>
```

Comment:
Can you use unordered list (ol) for the two options? I think that would separate the two options better from the rest.